static/js/components/LearnerSearch.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {
+  SearchkitComponent,
+  Hits,
+  NoHits,
+  MenuFilter,
+  SelectedFilters,
+  RefinementListFilter,
+  HitsStats,
+  Pagination,
+} from 'searchkit';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import Card from 'react-mdl/lib/Card/Card';
+import Dropdown from 'react-dropdown';
+
+import LearnerResult from './search/LearnerResult';
+import CountryRefinementOption from './search/CountryRefinementOption';
+import FilterVisibilityToggle from './search/FilterVisibilityToggle';
+import HitsCount from './search/HitsCount';
+import type { Option } from '../flow/generalTypes';
+
+export default class LearnerSearch extends SearchkitComponent {
+  props: {
+    checkFilterVisibility:  (s: string) => boolean,
+    setFilterVisibility:    (s: string, v: boolean) => void,
+  };
+
+  dropdownOptions: Option[] = [
+    { value: 'name', label: "Sort: name" },
+    { value: 'dob', label: "Sort: dob" },
+  ];
+
+  profileFieldOptions: Function = (): Object => ({
+    type: 'nested',
+    options: {
+      path: 'profile'
+    }
+  });
```

Comment:
seems like this doesn't need to be defined as a function, but i guess this is going away when we implement the search API endpoint